debian/changelog

```diff
@@ -0,0 +1,12 @@
+eos-data-distribution (0.1.0) UNRELEASED; urgency=medium
```

Comment:
Out of interest, why keep `master` and `debian-master` separate? This is our project: we know that it’s going to be built into a Debian package. Might as well save some rebase and branching faff and keep `debian/` in `master`, right?